src/solvers/dgsem_tree/dg_1d_parabolic.jl

```diff
@@ -204,6 +204,44 @@ function calc_interface_flux!(surface_flux_values,
     return nothing
 end
 
+# This is the version used when calculating the divergence of the viscous fluxes
+function calc_interface_flux!(surface_flux_values,
+                              mesh::TreeMesh{1}, equations_parabolic,
+                              dg::DG, parabolic_scheme::ViscousFormulationLocalDG,
+                              cache_parabolic)
+    @unpack neighbor_ids, orientations = cache_parabolic.interfaces
+
+    @threaded for interface in eachinterface(dg, cache_parabolic)
+        # Get neighboring elements
+        left_id = neighbor_ids[1, interface]
+        right_id = neighbor_ids[2, interface]
+
+        # Determine interface direction with respect to elements:
+        # orientation = 1: left -> 2, right -> 1
+        left_direction = 2 * orientations[interface]
+        right_direction = 2 * orientations[interface] - 1
+
+        # Get precomputed fluxes at interfaces
+        flux_ll, flux_rr = get_surface_node_vars(cache_parabolic.interfaces.u,
+                                                 equations_parabolic,
+                                                 dg, interface)
+
+        # Compute interface flux as mean of left/right viscous fluxes and an upwind term
+        flux = 0.5f0 * (flux_ll + flux_rr)
+        flux_jump = 0.5f0 * (flux_rr - flux_ll)
+
+        # Copy flux to left and right element storage
+        for v in eachvariable(equations_parabolic)
+            # Here, the flux is {{f}} + beta * [[f]], where beta is the LDG "switch", 
+            # which we set to  -1 on the left and +1 on the right in 1D.
+            surface_flux_values[v, left_direction, left_id] = flux[v] + flux_jump[v]
+            surface_flux_values[v, right_direction, right_id] = flux[v] - flux_jump[v]
```

Comment:
```suggestion
            # Here, the flux is {{f}} + beta * [[f]], where beta is the LDG "switch", 
            # which we set to  -1 on the left and +1 on the right in 1D.
            # This is equivalent to
            # - flux_rr for the left_direction, left_id
            # - flux_ll for the right_direction, right_id
            surface_flux_values[v, left_direction, left_id] = flux[v] + flux_jump[v]
            surface_flux_values[v, right_direction, right_id] = flux[v] - flux_jump[v]
```

Reply:
Is this really what we want? I would have expected to use the left values in one part (e.g., divergence) and the right values in the other part (e.g., gradient). In this case, we could use, for example,

```suggestion
            # Here, the flux is {{f}} + beta * [[f]], where beta is the LDG "switch", 
            # which we set to  -1 on the left and +1 on the right in 1D.
            # This is equivalent to flux_rr.
            val = flux[v] + flux_jump[v]
            surface_flux_values[v, left_direction, left_id] = val
            surface_flux_values[v, right_direction, right_id] = val
```

We still want to use a uniquely defined numerical flux per interface, don't we?

Reply:
Thanks! That's a great description. 

I didn't want to explicitly code in the `flux_rr` for left direction, etc because I thought the jump formulation was mathematically clearer, but I'll add these comments in all the implementations.

Reply:
@jlchan Sorry, I added another comment to this thread (see https://github.com/trixi-framework/Trixi.jl/pull/2333#discussion_r2021280060)

Reply:
Yes - I think I made a mistake with the `TreeMesh` implementation? 

For my own `DGMulti`-like solvers, I usually implement the LDG gradient/divergence as `u_avg +/- beta * u_jump`, which assumes that both `beta` and `u_jump` change sign across an element interface. However, since `DGSEM` solvers process the flux on each interface uniquely, `flux_jump` won't change sign appropriately and we don't end up with a single-valued flux.